pom.xml

```diff
@@ -122,6 +122,20 @@
             <version>3.1.2</version>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.junit</groupId>
+            <artifactId>junit-bom</artifactId>
+            <version>5.5.2</version>
```

Comment:
I was running 5.4.2, I will change to this more up to date version. No action needed